donate.html

```diff
@@ -143,108 +143,110 @@ <h1 class="hero-title" style="color: #000;"> Donating to </h1>
-            <p> <b> We promise to respect your donation by only using donor funds to support our mission of reaching gender equity in tech. </b> The most high cost ways we will spend funds this year are mailing supplies to hackers, providing supplies for mini activities all weekend long, and ensuring the virtual Technica experience is still amazing through our custom virtual platform.</p>
-            <p> <b> Additionally, Technica is a 501(c)(3) public charity, so your donations are tax-deductible.</b></p>
+  <div role="main">
+    <!-- Donor Promise -->
+    <div class="donate-promise">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-6 order-1 order-md-1 promise-text">
+            <!-- Header -->
+            <h1 class="our-donor" style="color: #000;">Our Donor Promise</h1>
+              <!-- Body -->
+            <div class="promise-body" style="text-align: left;">
+              <p> <b> We promise to respect your donation by only using donor funds to support our mission of reaching gender equity in tech. </b> The most high cost ways we will spend funds this year are mailing supplies to hackers, providing supplies for mini activities all weekend long, and ensuring the virtual Technica experience is still amazing through our custom virtual platform.</p>
+              <p> <b> Additionally, Technica is a 501(c)(3) public charity, so your donations are tax-deductible.</b></p>
+            </div>
           </div>
-        </div>
-        <div class="col-md-6 order-2 order-md-2" style="display:flex">
-          <div class="promise-image">
-              <img src="img/donor-promise.jpg" alt="Two happy people looking at a laptop together at Technica." class="center-block" style="margin-top: 5.5%; border-radius: 25px;">
+          <div class="col-md-6 order-2 order-md-2" style="display:flex">
+            <div class="promise-image">
+                <img src="img/donor-promise.jpg" alt="Two happy people looking at a laptop together at Technica." class="center-block" style="margin-top: 5.5%; border-radius: 25px;">
+            </div>
           </div>
         </div>
-      </div>
+        </div>
       </div>
     </div>
-  </div>
 
-  <!-- donor impact -->
-  <div class="donate-impact">
-    <div class="container text-center">
-      <div>
-        <h1 class="impact-header">Your Donation's Impact</h1>
-      </div>
-      <div class="row align-items-center justify-content-center">
-        <div class="col-sm">
-          <div class="donate-option1">
-              <h2 class="donate-option1-header">$50</h2>
-              <p>The cost to provide hardware kits to two hackers participating in our hardware track</p>
-          </div>
+    <!-- donor impact -->
+    <div class="donate-impact">
+      <div class="container text-center">
+        <div>
+          <h1 class="impact-header">Your Donation's Impact</h1>
         </div>
         <div class="col-sm">
           <div class="donate-option2">
```

Comment:
this should be donate-option1 for the different color, also looks like mobile media query is being applied on all screen sizes:
![Screen Shot 2021-07-13 at 5 17 01 PM](https://user-images.githubusercontent.com/62481939/125526413-75aac4b1-9055-46f6-a801-190540bbd786.png)